packages/core/r4b/data-types/identifier.ts

```diff
@@ -233,3 +266,13 @@ export const DEFAULT_SYSTEMS_LABELS: Required<
   "http://hl7.org/fhir/sid/us-npi": "NPI",
   "https://www.gs1.org/gtin": "GTIN",
 };
+
+/**
+ * Default values for `systemsPatterns`.
+ */
+export const DEFAULT_SYSTEMS_PATTERNS: Required<Record<string, string>> = {
+  "http://hl7.org/fhir/sid/us-ssn": "999-99-999",
+  "http://hl7.org/fhir/sid/us-mbi": "9a*9-a*9-aa99",
+  "https://fhir.infoway-inforoute.ca/NamingSystem/ca-social-insurance-number":
+    "999-999-999",
+};
```

Comment:
We should remove the Canadian SIN as it's not part of the official spec.
```suggestion
export const DEFAULT_SYSTEMS_PATTERNS: Required<Record<string, string>> = {
  "http://hl7.org/fhir/sid/us-ssn": "999-99-999",
  "http://hl7.org/fhir/sid/us-mbi": "9a*9-a*9-aa99",
};
```

Reply:
CA SIN removed.